tests/xharness/BCLTestInfo.cs

```diff
@@ -124,6 +124,8 @@ public virtual void Convert ()
 		{
 			var testName = TestName == "mscorlib" ? "corlib" : TestName;
 			var main_test_sources = Path.Combine (MonoPath, "mcs", "class", testName, testName + "_test.dll.sources");
+			if (!File.Exists (main_test_sources)) // TODO: revert this
+				return;
```

Comment:
???

Reply:
this is needed so this code doesn't freak out because the _test.dll.sources isn't there anymore. the test projects don't actually use it anymore afaik, it's just something @mandel-macaque needs to fix.

Reply:
All these classes are going away as soon as we remove the two PRs that remove the lasts tests.